code/game/objects/structures/barsign.dm

```diff
@@ -386,29 +363,20 @@
 	desc = "Open since 2125, Not much has changed since then; the engineers still release the singulo and the damn miners still are more likely to cave your face in that deliver ores."
 
 /datum/barsign/hiddensigns
```

Comment:
Is this subclass necessary? If it's just setting one new variable, why not have the variable set on the `/barsign` class and default it to FALSE and set it to true on each of the "hiddensigns"?